### PR TITLE
Added member photo/info for Kainoa Nishida

### LINKED ISSUE #290: Add member image for Kainoa Nishida

### PR DESCRIPTION
**Pull request by ctc-devops**
Authors:
 ### What does this PR contain?
This PR contains my photo and information for the website.
 ### How did you test these changes?
I tested these changes by running NPM Start
 ### Attach images (if applicable)
 Closes #290

### CODE REVIEW

**Comment by alanchangxyz** [comment]
not sure if this is actually ready or not but looks like no changes have been pushed yet, make sure you've got your local version of your branch and the github version synced up with `git push` (git will give you additional instructions if you've never pushed the branch to github before)

**Comment by KainoaNishida** [comment]
> not sure if this is actually ready or not but looks like no changes have been pushed yet, make sure you've got your local version of your branch and the github version synced up with `git push` (git will give you additional instructions if you've never pushed the branch to github before)

> okay I think I did it! my local branch should be pushed to the github version.

File: website/src/app/views/our-team/members.js [hunk 1]
```diff
@@ -224,6 +225,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/subin-kim-bba754180/',
         imageURL: subin,
       },
+      {
```
Comment (claudeisakeeb) [inline website/src/app/views/our-team/members.js:228]: Members should be displayed in ascending alphabetical order. Could you move your picture to the right spot?

File: website/src/app/views/our-team/members.js [hunk 2]
```diff
@@ -39,6 +39,7 @@ import orviela from '../../../images/ourTeam/memberImages/orviela.jpg';
 import dang from '../../../images/ourTeam/memberImages/dang.jpg';
 import raymond from '../../../images/ourTeam/memberImages/raymond.jpg';
 import subin from '../../../images/ourTeam/memberImages/subin.jpg';
+import kainoa from '../../../images/ourTeam/memberImages/kainoa.jpeg';
```
Comment (claudeisakeeb) [inline website/src/app/views/our-team/members.js:42]: Could you delete the unused "kainoa.jpg"?

**Review by claudeisakeeb — CHANGES_REQUESTED** [review]
Changes look good, just 2 small issues and this PR should be good to go

File: website/src/app/views/our-team/members.js (hunk 2 above)
Comment (KainoaNishida) [reply website/src/app/views/our-team/members.js:42]: Okay! I went back and did some edits and hopefully its all good now

**Review by alanchangxyz — APPROVED** [review]
as the tech bros say, LGTM

**Pull request merged**